go.mod

```diff
@@ -193,12 +194,14 @@ require (
 )
 
 replace (
+	// https://github.com/cometbft/cometbft-db/pull/42
+	github.com/cometbft/cometbft-db => github.com/yihuang/cometbft-db v0.7.1-0.20230306024652-3af958bced3e
```

Comment:
https://github.com/cometbft/cometbft-db/pull/42

Reply:
forked here: https://github.com/crypto-org-chain/cometbft-db if expected cronos PR needs to be merged before the upstream cometbft-db patch

Reply:
let's use fork first

Reply:
Just added v1.0.5 prepare directly in this PR.